CorsixTH/Lua/hospitals/player_hospital.lua

```diff
@@ -382,6 +415,9 @@ function PlayerHospital:onEndMonth()
   -- Advise the player on cash flow.
   if self:hasStaffedDesk() then
     self:monthlyAdviceChecks()
+  else
+    local today = self.world:date()
+    self:checkReceptionAdvice(today:monthOfYear(), today:year())
```

Comment:
The function doesn't do anything if you have a staffed desk, so it already takes care of the pre-condition by itself. No need to do that here as well.